api/lib/infrastructure/repositories/tutorial-evaluation-repository.js

```diff
@@ -0,0 +1,22 @@
+const BookshelfTutorialEvaluation = require('../data/tutorial-evaluation');
+
+module.exports = {
+  async addEvaluation({ userId, tutorialId }) {
+    const foundTutorialEvaluation = await BookshelfTutorialEvaluation.where({ userId, tutorialId }).fetch();
+    if (foundTutorialEvaluation) {
+      return _toDomain(foundTutorialEvaluation);
```

Comment:
J'ai cru comprendre cet [utilitaire](https://github.com/1024pix/pix/blob/dev/api/lib/infrastructure/utils/bookshelf-to-domain-converter.js) pour faire les conversions en objets du domaine. Est ce bien le cas ?

Reply:
Effectivement, on utilise cet utilitaire lorsqu'on souhaite matérialiser un objet du domain grâce à son modèle de données. Ici, nous ne voulons pas de modèle de données parce qu'on n'utilise l'information que côté API. C'est ce qui donnera naissance aux Read Objects dans un futur proche apparemment.